src/shared/components/FileUpload/index.tsx

```diff
@@ -74,15 +74,18 @@ const FileUploader: React.FunctionComponent<FileUploaderProps> = ({
   >([]);
   const [fileList, setFileList] = React.useState<UploadFile[]>([]);
 
-  const handleFileUpload = async (customFileRequest: CustomFileRequest) => {
-    try {
-      const options = storageId ? { storage: storageId } : undefined;
-      const nexusFile = await onFileUpload(customFileRequest.file, options);
-      setRecentlyUploadFileList([...recentlyUploadedFileList, nexusFile]);
-      customFileRequest.onSuccess('Successfully uploaded file');
-    } catch (error) {
-      customFileRequest.onError(error);
-    }
+  // The types for this are wrong :(
+  // https://github.com/ant-design/ant-design/blob/master/components/upload/interface.tsx#L79
```

Comment:
Did we file a bug report?

Reply:
How is that related to the new menu PR?

Reply:
its not, but I couldn't get it to compile otherwise